### PR TITLE
Fix InputFile::fromFile

### PR DESCRIPTION
**Pull request by roynatech2544**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -11,7 +11,7 @@ option(BUILD_SHARED_LIBS "Build tgbot-cpp shared/static library." OFF)
 option(BUILD_DOCUMENTATION "Build doxygen API documentation." OFF)
 
 # sources
-set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD 17)
```
Comment (JellyBrick) [inline CMakeLists.txt:14]: It seems like a breaking change, is there any way to work around it without changing 14 to 17?

Comment (roynatech2544) [reply CMakeLists.txt:14]: What do you mean by breaking? It passed the CI, however there is boost::filesystem for that anyways

**Comment by roynatech2544** [comment]
I just used the boost variant now

**Review by reo7sp — APPROVED** [review]
Thanks

**Pull request merged**